manifests/config/resource_source.pp

```diff
@@ -80,6 +80,10 @@
   Boolean $url_cache                                             = $rundeck::params::url_cache,
   Integer $url_timeout                                           = $rundeck::params::url_timeout,
   Boolean $use_default_mapping                                   = true,
+  String $endpoint_url                                           = '',
```

Comment:
Datatypes \o/

Reply:
Is it good that this defaults to empty string and not undef?

Reply:
Yes, it is better to put undef